subsys/net/lib/nrf_cloud/coap/cddl/nrf_cloud_coap_ground_fix.cddl

```diff
@@ -31,7 +31,7 @@ cell = {
 lte_ar = [1*5 cell]
 
 ap = {
-	macAddress => bstr,
+	macAddress => tstr,
```

Comment:
If we as the backend team to change the REST API, I am wondering how the MAC address on the ap object would need to change on the REST interface to support the bstr format instead of tstr?  Will your automatic conversion of a 6 byte bstr  be the decimal equivalent of a 6 byte binary value?  For example, the device converts the MAC address 94-E6-F7-E9-5D-2C to a 6 byte bstr, and your proxy then converts that to the JSON "macAddress":163719722655024?

Reply:
So `bstr` is converted to json byte array which is represented as base64 so it would be:
```
"macAddress":"lOb36V0s"
```

If we define cddl as (it should be the same size as bstr):
```
macAddress => uint
```

Then REST API could use:
```
"macAddress":163719722655020
```

I believe for REST API second option will be easier to parse, either string or number which is then decoded into mac address.

Reply:
I discussed this with Cole and Lincoln, and they do not feel that this uint option is the right approach.  They would need to implement their own conversion from uint back to a string of colon-separated ascii hex bytes because that is the format all the location services vendors want, which means this just moves any opportunity for human error from you to them.  Also, it would be confusing to customers to offer both strings and uints, and would be a support problem.

Please implement the conversion from bstr to string using glue logic.  I know you don't like that answer, but it is the most efficient for the device power and data usage, and keeps the REST API easy to understand and use.  We do not expect customers to implement the CBOR conversions themselves.

Reply:
Well, I would say exact same concerns as Core ;). I hope we can still thing about it.